src/common/styles/utilities.tsx

```diff
@@ -174,7 +175,9 @@ export const BannerContentWrapper: FC<
   return (
     <BannerWrapper bannerShowing={bannerShowing}>
       <StagingBanner>
-        You are currently on the <b>staging</b> server.
+        <Trans ns='common' i18nKey="stagingBanner">
+          You are currently on the <strong>staging</strong> server.
```

Comment:
The `<Trans>` component does not support the bold tag by default, but it does support `<strong>` so I used that here and in other places. This text is also replaced by the translation file content.